networkx/algorithms/isomorphism/tests/test_isomorphism.py

```diff
@@ -1,5 +1,8 @@
+import pytest
+
 import networkx as nx
 from networkx.algorithms import isomorphism as iso
+from networkx.exception import NetworkXError
```

Comment:
This import isn't necessary since the exception is being called from the `nx` namespace below

```suggestion
```